lib/perl/Genome/Qc/Tool/Bedtools/Intersect.pm

```diff
@@ -0,0 +1,35 @@
+package Genome::Qc::Tool::Bedtools::Intersect;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::Qc::Tool::Bedtools::Intersect {
+    is => 'Genome::Qc::Tool::WithVariationListVcf',
+};
+
+sub cmd_line {
+    my $self = shift;
+
+    my @cmd = qw(gmt bed-tools intersect);
+    while (my ($input, $value) = each %{$self->gmt_params}) {
+        if ($input eq 'header') {
+            push @cmd, "--$input";
+        }
+        else {
+            push @cmd, "--$input=$value";
+        }
+    }
+    return @cmd;
+}
+
+sub get_metrics {
+    return ();
+}
+
+sub target_region_bed_file {
```

Comment:
This PR doesn't implement the QC config changes necessary to run verify bam id (since the exact setup isn't clear yet). Bedtools intersect will be used for capture data which the `target_region_bed_file` as `input-file-b`.